### PR TITLE
FX: ZENKO-1112 Management client error logging

### PR DESCRIPTION
**Pull request by nicolas2bert**

### CODE REVIEW

**Comment by nicolas2bert** [comment]
@bert-e help

**Comment by nicolas2bert** [comment]
@bert-e reset

**Review by rachedbenmustapha — CHANGES_REQUESTED** [review]
I think we should reshape these errors as well in the top-level logs from initManagement, as well as the apply overlay/save overlay functions.

**Review by tmacro — APPROVED** [review]

File: lib/management/push.js [hunk 1]
```diff
@@ -77,9 +79,10 @@ function createWSAgent(pushEndpoint, env, log) {
  */
 function startWSManagementClient(pushEndpoint, instanceId, token) {
     logger.info('connecting to push server');
-    function _logError(error) {
+    function _logError(error, errorMessage, method) {
         if (error) {
-            logger.error('management client error', { error });
+            logger.error(`management client error: ${errorMessage}`,
```
Comment (jeremyds) [inline lib/management/push.js:84]: no need to reshape here?

**Review by jeremyds — COMMENTED** [review]
Except my comment, the rest lgtm

**Review by JianqinWang — APPROVED** [review]

**Review by rachedbenmustapha — APPROVED** [review]

**Comment by rahulreddy** [comment]
@bert-e reset

**Pull request merged**